exts/base.py

```diff
@@ -177,7 +178,7 @@ def __init__(self, bot: commands.Bot):
 
     @commands.Cog.listener()
     async def on_ready(self):
-        if not self.bot_invite:
+        if not self.bot_invite and self.bot.user:
```

Comment:
How is is possible for the bot to have no user? QRM being a bot based on traditional text commands, this should not be possible.